src/lib/services/auth/service/oauth2/token.ts

```diff
@@ -1,7 +1,21 @@
 import { ID } from '../../config.js';
 import { HTTP_METHODS, serviceRequest } from '../../../../services/shared.js';
 
-import type { ServiceMethod } from '../../../types.js';
+import type { ServiceMethod, ServiceMethodOptions } from '../../../types.js';
+
+/**
+ * Format and inject properties that are specific to the `/token` resources.
+ */
+function injectServiceOptions(options: ServiceMethodOptions): ServiceMethodOptions {
+  return {
+    ...options,
+    body: new URLSearchParams(options?.payload as Record<string, string>).toString(),
+    headers: {
+      ...(options?.headers || {}),
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+  };
+}
 
 /**
  * Token Introspection
```

Comment:
Do we need to support this endpoint? I thought this was only meant to be used by resource servers?

Reply:
It is, but I went ahead and added it for our server-side JavaScript friends (i.e. Node.js users).